### DISCUSSION TITLE
How to get exceptionMessage in click house query?

### DISCUSSION THREAD

**Discussion by TheShubhendra** [SigNoz/signoz#4741] [Q&A]
I have setup a alert channel to discord. When any exception occurs in my python web servers I want to get stacktrace in my discord channel. But i am not able to get exceptionMessage in click house query builder
```
SELECT 
	count() as value,
	toStartOfInterval(timestamp, toIntervalMinute(1)) AS interval,
	exceptionMessage
FROM signoz_traces.signoz_error_index_v2
WHERE exceptionType !='OSError'
AND timestamp BETWEEN {{.start_datetime}} AND {{.end_datetime}}
GROUP BY serviceName, interval;
```
This is giving me the error

**Comment by srikanthccv** [comment]
This is an invalid query. The `exceptionMessage` should be part of the group by clause.

**Comment by TheShubhendra** [comment]
> This is an invalid query. The `exceptionMessage` should be part of the select clause.

Sorry, I didn't get that. the one i shared , Isn't it part of select clause only ?

**Comment by srikanthccv** [comment]
I meant group by clause

**Reply by TheShubhendra** [reply]
Thanks! The query is working now but how can I inject the exceptionMessage in the alert description. 
I tried setting description to `{{$exceptionMessage}}` but it didn't work
